cron.go

```diff
@@ -0,0 +1,41 @@
+package webhook
+
+import (
+	"time"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+// Cron type for cron updates
+type Cron struct {
+	k8sClient    kubernetes.Interface
+	syncInterval time.Duration
+	log          Logger
+	cache        *Cache
+}
+
+// NewCron - creates new cron object
+func NewCron(k8sClient kubernetes.Interface, syncInterval time.Duration, cache *Cache, log Logger) *Cron {
+	return &Cron{
+		k8sClient:    k8sClient,
+		syncInterval: syncInterval,
+		log:          log,
+		cache:        cache,
+	}
+}
+
+// FullResync - add all namespaces to the queue for full resync
+func (c *Cron) HourlySync(stopCh <-chan struct{}) {
```

Comment:
Can we rename this function as it may not be hourly depending on the sync interval?